app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="<%= dom_id food %>">
+  <% if food.user == current_user %>
+    <tr>
+      <td><%= food.name%></td>
+      <td><%= food.measurement_unit %></td>
+      <td>$<%= food.price %></td>
+      <td><%= button_to "Delete", food, method: :delete, class:"btn btn-outline-danger btn-sm" %></td>
+    </tr>
+  <% end %>
```

Comment:
- [ ] It's a requirement that you follow the given wireframe for the `foods` list. Kindly realign the `Add Food` button as in the provided wireframe.

Provided Wireframe
![image](https://user-images.githubusercontent.com/81932478/187462494-fc0634d6-25e6-4370-bd32-cf2ff0b575ca.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/187462603-8fe2f3c6-4254-45c2-9450-23e6b15d73ca.png)